DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/annotated_class_clean_coding/src/annotated_coding.cpp

```diff
@@ -0,0 +1,140 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+#include "annotated_class_util.hpp"
+#include "exception_handler.hpp"
+
+constexpr int kBL1 = 1;
+constexpr int kAlignment = 32;
+constexpr int kWidth = 256;
+
+// This type alias groups together all the properties used by the `a` pointer,
+// and can be re-used in the annotated memory allocation in the host code.
+// This syntax works if you have added the `-std=c++20` flag to your compiler
+// command.
+using annotated_arg_t = sycl::ext::oneapi::experimental::annotated_arg<
+    int *, fpga_tools::properties_t<
+               sycl::ext::intel::experimental::buffer_location<kBL1>,
+               sycl::ext::intel::experimental::dwidth<kWidth>,
+               sycl::ext::oneapi::experimental::alignment<kAlignment>>>;
+
+struct MyIP {
+  annotated_arg_t a;
+  int size;
+
+  void operator()() const {
+#pragma unroll 8
+    for (int i = 0; i < size; i++) {
+      a[i] *= 2;
+    }
+  }
+};
+
+bool checkResult(int *arr, int size) {
```

Comment:
```suggestion
bool CheckResult(int *arr, int size) {
```
Code sample code standards request that function names be in `PascalCase()`

(I forgot about this until today, I should have mentioned this sooner)